pkgs/slurm-ops/src/slurm_ops/core/base.py

```diff
@@ -517,6 +517,11 @@ def path(self) -> Path:
         """Get the path to the `jwt_hs256.key` secret file."""
         return self._file
 
+    @path.setter
+    def path(self, new_path: str) -> None:
+        """Set the path to the `jwt_hs256.key` secret file."""
+        self._file = Path(new_path)
```

Comment:
Since the JWT key is no longer under `/var/lib/slurm/checkpoint`, let's remove the setters here for the path to the file. 

While we _might_ potentially run into this situation again where we need to modify the key path, we're not doing it now, so we should remove it because:

1. It's "dead code". It provides additional functionality that isn't actually used.
2. We need to test this otherwise it will impact our coverage scores. No point spending pulse hours on this if it isn't used at all :sweat_smile: 

```suggestion
```